### DISCUSSION TITLE
Block Widgets

### DISCUSSION THREAD

**Discussion by digidoda** [bootscore/bootscore#632] [Q&A]
Hi there,

Is it possible to reinstate the block widgets when using bootscore? I'm looking at building a site with Sensei LMS and a lot of their functionality is using blocks so wondered if it would be a good fit?

**Comment by crftwrk** [comment]
Hello,

for now, you can only remove those lines in main-theme:

https://github.com/bootscore/bootscore/blob/aaff27960b92aa6186e5770aab5a989eac7538e6/inc/widgets.php#L17-L23

Of course, this will be overridden by next updates, so you have to keep this in mind. We will enable Gutenberg widgets in theme if there is an acceptable solution for search widget.
